frontend/nuxt.config.js

```diff
@@ -8,7 +8,7 @@ export default {
 
   // Global page headers: https://go.nuxtjs.dev/config-head
   head: {
-    title: 'PolkaStats | Polkadot block explorer',
+    title: 'Cere Network Stats',
```

Comment:
Let's use `Cerebellum` instead of `Cere` in title here and everywhere.

Reply:
Replaced